test/posix_test.dart

```diff
@@ -32,8 +32,6 @@ void main() {
     expect(context.extension('a.b/c.d', 2), '.d');
     expect(() => context.extension(r'foo.bar.dart.js', 0), throwsRangeError);
     expect(() => context.extension(r'foo.bar.dart.js', -1), throwsRangeError);
-    expect(
```

Comment:
`null` is no longer allowed

Reply:
This is a place where we might want to add a `posix_unsound_null_test.dart`... Doesn't need to happen now.